tests/Browser/ReviewPostTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Tests\Browser;
+
+use Rogue\Models\Post;
+use Tests\DuskTestCase;
+use Rogue\Models\Signup;
+use Laravel\Dusk\Browser;
+use Tests\Browser\Pages\HomePage;
+use Tests\Browser\Pages\CampaignInboxPage;
+use Tests\Browser\Pages\CampaignSinglePage;
+
+class ReviewPostTest extends DuskTestCase
+{
+    /**
+     * Test user flow of approving a pending Post and tagging it as a Good Photo.
+     *
+     * @group ReviewPost
+     * @return void
+     */
+    public function testApprovingAndTaggingAPost()
+    {
+        // Create a signup and an associated post with a 'pending' status
+        // so there will be a post in the campaign inbox.
+        $signup = factory(Signup::class)->create();
+        $post = $this->createAssociatedPostWithStatus($signup, 'pending');
+
+        $this->browse(function (Browser $browser) use ($signup) {
+            $browser->visit(new HomePage)
+                    ->click('@login-button')
```

Comment:
Maybe we can make a function on DuskTestCase that you pass the browser to? Would be nice to DRY this up.

```php
// DuskTestCase

public function login(Browser $browser)
{
    $browser->visit(new HomePage)
            ->click('@login-button')
            ->assertPathIs('/register')
            ->clickLink('Log In')
            ->type('username', env('NORTHSTAR_EMAIL'))
            ->type('password', env('NORTHSTAR_PASSWORD'))
            ->press('Log In');
}

// ReviewPostTest
public function testApprovingAndTaggingAPost()
{
    $this->browse(function (Browser $browser) use ($signup) {
        $this->login($browser);
       
        $browser->assertPathIs('/campaigns')
                ->visit('/campaigns/' . $signup->campaign_id)
                // and so on...
```

Reply:
I think we could also assert login status here with `$browser->assertAuthenticated()`, so that we could always call `$this->login($browser)` and ensure that we're logged in regardless of when we run a test.